crates/path_trie/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "path_trie"
```

Comment:
Does this name still make sense? It seems very much tailored towards unclobbering

Reply:
I changed name to path resolver.